src/frontend/web_application/src/scenes/Contact/presenter.jsx

```diff
@@ -78,58 +103,136 @@ class Contact extends Component {
+        >Cancel</Button>
+        <TextBlock className="s-contact__bar-title">
+          {__('contact.edit_contact.title')}
+        </TextBlock>
+        <Button
+          onClick={this.toggleEditMode} // FIXME: this should validate contact change
+          responsive="icon-only"
+          icon="check"
+          className="s-contact__action"
+        >Validate</Button>
+      </div>
+    );
+  }
+
+  renderActionBar = () => {
+    const { contact } = this.props;
+
+    return (
+      <div className="s-contact__action-bar">
+        <TextBlock className="s-contact__bar-title">
+          {contact.title}
+        </TextBlock>
+        <DropdownControl
+          toggle={this.dropdownId}
+          className="s-contact__actions-switcher float-right"
+          icon="ellipsis-v"
+        />
+
+        <DropdownMenu
+          id={this.dropdownId}
+          className="s-contact__actions-menu"
+          position="bottom"
+          closeOnClick
+        >
+          <VerticalMenu>
+            <VerticalMenuItem>
+              <Button
+                onClick={this.toggleEditMode}
+                className="s-contact__action"
+                display="expanded"
+              >Edit</Button>
+            </VerticalMenuItem>
+            <VerticalMenuItem>
+              <Button
+                onClick={this.openTagsModal}
+                className="s-contact__action"
+                display="expanded"
+              >Manage Tags</Button>
+              {
+                // FIXME: when you open tagsModal, and then close it,
+                // Dropdown bugs.
+                this.renderTagsModal()
+              }
+            </VerticalMenuItem>
+            <VerticalMenuItem>
+              <Button
+                onClick={this.openTagsModal}
+                className="s-contact__action"
+                display="expanded"
+              >Share</Button>
+            </VerticalMenuItem>
+            <VerticalMenuItem>
+              <Button
+                onClick={this.handleContactDelete}
+                className="s-contact__action"
+                display="expanded"
+              >Delete</Button>
+            </VerticalMenuItem>
+          </VerticalMenu>
+        </DropdownMenu>
+      </div>
+    );
+  }
+
   render() {
     const { __, isFetching, contact } = this.props;
 
     return (
       <div>
         {contact && (
           <MenuBar className="s-contact__menu-bar">
-            <Button
-              onClick={this.handleContactDelete}
-              responsive="icon-only"
-              icon="trash"
-            >{__('contact.action.delete_contact')}</Button>
-            <Button
-              onClick={this.handleClickTags}
-              responsive="icon-only"
-              icon="tags"
-            >{__('contact.action.edit_tags')}</Button>
-            {this.renderTagsModal()}
+            {
+              // FIXME: edit and action bars be displayed in fixed Header,
+              // not in MenuBar
```

Comment:
the simplest way would be to create a FixedBar comp. with z-index > to Header's z-index.
another way would be to display this FixedBar instead of Header but this seems harder :/